.github/pull_request_template.md

```diff
@@ -1,15 +1,24 @@
-### Description
+## 🔗 Related Links
 
-_Describe why we made this change and what was changed_
+- [ ] [ClickUp](https://app.clickup.com/t/XXXXXX)
+- [ ] [Figma](https://www.figma.com/file/XXXXXX)
 
-### Demo
+## ⬅️ As Is
 
-_Add relevant screenshots or demo GIFs_
+Explain the current situation of the code
 
-| Before | After |
-| --- | --- |
-| *img or gif* | *img or gif* |
+- Example1) New SGI and custom item flow required but don't have this implementation yet
+- Example2) No basic inpage for new client
 
-### Ticket
+## ➡️ To Be
 
-_Ticket link, if applicable_
\ No newline at end of file
+- [ ] Example1) Create SGI and custom item for new client
+- [ ] Example2) Create basic inpage for new client with hardcoding for external product ID
+
+## ☑️ Checklist
+
+- [ ] Useless comments and/or `Log.d` etc are **removed**
+- [ ] Unit test are **covered**
+- [ ] Code has been **deployed and tested** on STG
+- [ ] ClickUp ticket status has been **updated** to `production`
+- [ ] Update CHANGELOG.md with the new changes
```

Comment:
This is included as I'm not sure how to merge this to trunk without approval, but I need the change to be applied to trunk.